person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age, :parent_permission
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] It would be nice to have a setter for rentals to follow the project requirements to form the many-to-many relationship completely 👍 So it would be better if you if you have `attr_accessor :rentals` 😊